keiko-core/src/main/kotlin/com/netflix/spinnaker/q/QueueProcessor.kt

```diff
@@ -82,8 +85,17 @@ class QueueProcessor(
             }
           }
         } catch (e: RejectedExecutionException) {
-          log.warn("Executor at capacity, immediately re-queuing message", e)
-          queue.push(message)
+          val requeueDelay = Duration.ofSeconds(2 + random.nextInt(5).toLong())
```

Comment:
Isn't there a `nextLong`?

Reply:
No, I just looked and there isn't (at least not one with a parameter). Objection withdrawn.

Reply:
👍